nixos/modules/services/web-apps/mediawiki.nix

```diff
@@ -205,6 +204,12 @@ in
 
       package = mkPackageOption pkgs "mediawiki" { };
 
+      # https://www.mediawiki.org/wiki/Compatibility
+      phpPackage = (mkPackageOption pkgs "php" { }) // {
+        default = if versionAtLeast cfg.package.version "1.42.0" then pkgs.php82 else pkgs.php81;
+        defaultText = literalExpression ''if versionAtLeast cfg.package.version "1.42.0" then pkgs.php82 else pkgs.php81'';
```

Comment:
cfg here should be replaced by config.services.mediawiki

Reply:
Fixed. Thanks!